urnai/environments/environment_base.py

```diff
@@ -11,9 +10,9 @@ class EnvironmentBase(ABC):
     should still be used as a wrapper (e.g. implementing an environment for OpenAI gym).
     """
 
-    def __init__(self, id: str, render=False, reset_done=True):
-        self.id = id
-        self.render = render
+    def __init__(self, map_name: str, visualize=False, reset_done=True):
```

Comment:
```suggestion
    def __init__(self, map_name: str, visualize: bool = False, reset_done: bool = True):
```

Reply:
I didn't know this could be done with default parameters :0
Should I do it with the other ones from other methods too?

Reply:
Yes!